contributing.md

```diff
@@ -387,21 +387,6 @@ will trigger workflows "On pull request*" (on Spark repo) that will look/watch f
 1. The related JIRA, if any, will be marked as "In Progress" and your pull request will 
 automatically be linked to it. There is no need to be the Assignee of the JIRA to work on it, 
 though you are welcome to comment that you have begun work.
-1. The Jenkins automatic pull request builder will test your changes
```

Comment:
Here we can write about enable github actions on fork. And that github actions are running test.